misc/python/materialize/scalability/workloads.py

```diff
@@ -64,3 +68,15 @@ def operations(self) -> list["Operation"]:
 class UpdateWorkload(Workload):
     def operations(self) -> list["Operation"]:
         return [Update()]
+
+
+class EstablishConnectionWorkload(WorkloadWithContext):
+    def amend_data_before_execution(self, data: OperationData) -> None:
+        data.push("endpoint", self.endpoint)
+        data.push("schema", self.schema)
+        data.remove("cursor")
+
+    def operations(self) -> list["Operation"]:
+        return [
+            OperationChainWithDataExchange([Connect(), SelectCount(), Disconnect()])
```

Comment:
I think `SelectOne` is more appropriate here, just so that it is 100% certain that neither dataflow nor storage is involved in this particular benchmark.

Reply:
Makes sense, adjusted. ✅